docker/1.11.0/final/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,64 @@
+FROM tensorflow/serving:1.11.0 as tensorflow_serving_image
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+ARG framework_installable
+ARG framework_support_installable=sagemaker_tensorflow_container-1.0.0.tar.gz
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    python \
+    curl \
+    nginx \
+ && rm -rf /var/lib/apt/lists/*
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
+
+# Removing tests to free some space
+RUN set -ex; \
+	curl -k -O https://bootstrap.pypa.io/get-pip.py && \
+	python get-pip.py \
+		--disable-pip-version-check \
+		--no-cache-dir \
+		"pip==18.1" \
+	; \
+	pip --version; \
+	find /usr/local -depth \
+		\( \
+			\( -type d -a \( -name test -o -name tests \) \) \
+			-o \
+			\( -type f -a \( -name '*.pyc' -o -name '*.pyo' \) \) \
+		\) -exec rm -rf '{}' +; \
+	rm -f get-pip.py
+
+WORKDIR /
+
+# Install TF Serving pkg
+COPY --from=tensorflow_serving_image /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
```

Comment:
Why is it better than installing it from repo like we were before:
https://github.com/aws/sagemaker-tensorflow-container/blob/master/docker/1.9.0/final/py2/Dockerfile.cpu#L54-L57